app/api/routers/breeze_buddy/daily/handlers.py

```
@@ -0,0 +1,124 @@
+        # 4. Prepare runner arguments for Daily transport
+        runner_args = DailyRunnerArguments(
+            room_url=room_url,
+            token=bot_token,
+            body={
+                "lead_id": request.lead_id,
+                "session_id": session_id,
+            },
+        )
+
+        # 5. Start the bot in background
+        asyncio.create_task(bot(runner_args))
```

Ruff (0.14.10)

Store a reference to the return value of `asyncio.create_task`

(RUF006)